sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-sources/coreos-sources-6.10.9.ebuild

```diff
@@ -36,6 +36,5 @@ IUSE=""
 # local patches overlap with the upstream patch.
 UNIPATCH_LIST="
 	${PATCH_DIR}/z0001-kbuild-derive-relative-path-for-srctree-from-CURDIR.patch \
-	${PATCH_DIR}/z0002-revert-pahole-flags.patch \
```

Comment:
Have you tested this?
When `pahole` is executed with `-j` (parallel) then btf metadata order is non-deterministic and the built kernel and modules don't match.

It doesn't have to be a revert, but we need to carry some patch (unless something significant changed).

Reply:
definitely, working on it. pahole flags moved to scripts/Makefile.btf, so that needs to be addressed, was working now on a patch.

Reply:
We recently updated pahole to a newer version ([1.27](https://git.kernel.org/pub/scm/devel/pahole/pahole.git/tag/?h=v1.27)) that was supposed to be reproducible regardless of how many threads it uses, but dropping the patch didn't work for me.

Reply:
Seems like we may still need a kernel patch to pass `--btf_features=all,reproducible_build`: https://git.kernel.org/pub/scm/devel/pahole/pahole.git/commit/?h=v1.27&id=43bd3efa85656565129063cdd6dd7499e44a7867

Reply:
this could be upstreamed

Reply:
I will test it asap and send it to LKML if it works.

Reply:
Added the reproducible_build flag to the pahole params, although I don't like how that addition is done, that file is a beautiful soup and needs some better management.